Sources/Down/AST/Visitors/AttributedStringVisitor.swift

```diff
@@ -20,17 +20,23 @@ public class AttributedStringVisitor {
 
     private let styler: Styler
     private let options: DownOptions
+    private let listPrefixGeneratorBuilder: ListItemPrefixGeneratorBuilder
```

Comment:
Perhaps `ListItemPrefixGeneratorBuilder` is not necessary if instead we use a closure:

```swift
private let listPrefixGeneratorBuilder: (List) -> ListItemPrefixGenerator
```

What do you think?

Reply:
I was not a fan of the builder neither. Will refactor based on your suggestion.